content/20.treatments.md

```diff
@@ -10,8 +10,14 @@ Additionally, there have been suggestions within the scientific community that n
 In the following sections, we critically appraise the literature surrounding the repurposing of existing treatments and development of novel therapeutics for the prevention, mitigation, and treatment of coronavirus infections.  
 
 ### Treatment of Symptoms
+The clinical picture of SARS-CoV-2 infection ranges from being asymptomatic, going through mild symptoms, such as cough, sore throat and fever, to severe pneumonia with respiratory failure which can lead to death. The development and the approval of a vaccine against this new microbe will require at least a year. Afterwards, the vaccine will face another challenge. It will in fact need to be produced in vast quantities in order to be distributed to all countries, while many companies do not have such production capacity. While at the moment the most effective way to fight Corona virus is social distancing, scientist are also focusing on other ways to cure the infection. Currently, treatments are going in two different directions. On one side, efforts are focusing on the development of new drugs which can cure the symptoms. On the other side, scientist are testing new antivirals that can inhibit the virus amplification in human bodies. In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 
+
+Tocilizumab 
+
+Background: A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (Zhou et al. 2020). This indicates an inflammatory response out of control or what is called “cytokine storm”. In this context IL-6 plays a key role (Tisoncik et al. 2012). IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation. In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis. It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001). The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 
```

Comment:
Can you please add newlines (with the "Enter" key) after each sentence?  That makes it easier to give feedback on the contribution and track changes.  This paragraph would look like:
```suggestion
Background: A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (Zhou et al. 2020).
This indicates an inflammatory response out of control or what is called “cytokine storm”.
In this context IL-6 plays a key role (Tisoncik et al. 2012).
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001).
The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 
```
You can click the button here to "Commit suggestion" to update this paragraph and then edit the others yourself.

For references, citing with the DOI is the most reliable option in Manubt.  See https://github.com/greenelab/covid19-review/pull/68#discussion_r398217044 for an example of how to get the DOI for a journal publication and cite it in the text.

Reply:
Hi @aadattoli , I added the line breaks (enters) between each sentence to make it easier for people to give you feedback. 
Like @agitter said, if you could switch your citations to the manubot format, it would be super helpful! For example, for Channappanavar & Perlman, 2017, instead it would be:
`[@doi:10.1007/s00281-017-0629-x]`
If you need any help figuring this out, please ask on the gitter chat and we can try to help!